corehq/apps/export/views.py

```diff
@@ -417,6 +417,30 @@ def get_export_schema(self, export_instance):
         )
 
 
+class DeleteNewCustomExportView(BaseNewExportView):
+    urlname = 'delete_new_custom_export'
+    http_method_names = ['post']
+    is_async = False
+
+    @property
+    def export_id(self):
+        return self.kwargs.get('export_id')
+
+    def commit(self, request):
+        self.export_type = self.kwargs.get('export_type')
+        export = self.export_instance_cls.get(self.export_id)
+        export.delete()
```

Comment:
Is there a check to verify that the user has permission to delete the export, or will it allow any (authenticated) user to delete any export for which they have a valid `export_id`?